### PR TITLE
Remove default about:blank load action in the  View constructor

### PR DESCRIPTION
**Pull request by psaavedra**
Signed-off-by: Pablo Saavedra <psaavedra@igalia.com>

### CODE REVIEW

File: core/cog-view.c [hunk 1]
```diff
@@ -103,10 +103,6 @@ static void
 cog_view_constructed (GObject *object)
 {
     G_OBJECT_CLASS (cog_view_parent_class)->constructed (object);
-
-    webkit_web_view_load_html (WEBKIT_WEB_VIEW (object),
-                               "<!DOCTYPE html><html><body></body></html>",
-                               "about:blank");
```
Comment (aperezdc) [inline core/cog-view.c:109]: Now that the `.constructed` callback does nothing, we can remove the whole `cog_view_constructed()` function.

**Review by aperezdc — CHANGES_REQUESTED** [review]
The idea sounds fine to me, but let's clean the code a bit more :smile:

File: core/cog-view.c (hunk 1 above)
Comment (psaavedra) [reply core/cog-view.c]: Updated. Please take in consideration these changes to be merged in Cog 0.10. Thanks.

Comment (aperezdc) [reply core/cog-view.c]: We don't need to do anything for 0.10 because `master` does not have the dummy load of `about:blank` :wink:

**Pull request merged**